kubernetes/ansible/roles/sunbird-monitoring/templates/prometheus-operator.yaml

```diff
@@ -11,10 +11,12 @@ fullnameOverride: sunbird-monitoring
 # Enabling external prometheus scrape config 
 prometheus:
   prometheusSpec:
-    additionalScrapeConfigsExternal: {{ additional_scrape_configs_enabled | d(true) }}
+    thanos:
+      version: {{ thanos_sidecar_version | d('v0.11.0') }}
+    additionalScrapeConfigsExternal: true
     retention: "{{ prometheus_retention_time | d('90d') }}"
     externalLabels:
-      cluster: "{{ kubernetes_cluster_name | default('kubernetes-1')}}"
+      sb_cluster: "{{ kubernetes_cluster_name | default('kubernetes-1')}}"
```

Comment:
This will require a dashboard change too. Hope its done below.

Reply:
Nginx dashboards are changed. Api dashboard, hope we had changed